CODEOWNERS

```diff
@@ -9,7 +9,7 @@
 # The format is described: https://github.blog/2017-07-06-introducing-code-owners/
 
 # These owners will be the default owners for everything in the repo.
-*       @amika-sq
+*       @KendallWeihe @nitro-neal @kirahsapong @shamilovtim @diehuxx @decentralgabe
```

Comment:
looks like an issue with @shamilovtim maybe he is missing write access to the repo